checks/raw/signed_releases.go

```diff
@@ -55,6 +55,24 @@ func SignedReleases(c *checker.CheckRequest) (checker.SignedReleasesData, error)
 		})
 	}
 
+	for _, v := range versions.Versions {
+		prov := checker.PackageProvenance{}
+
+		if len(v.SLSAProvenances) > 0 {
+			prov = checker.PackageProvenance{
```

Comment:
all the changes in this file are already in main I believe, so can you update your branch so they disappear from the diff?